### PR TITLE
prepare Community page for publishing AEROGEAR-1600

### PR DESCRIPTION
**Pull request by lfryc**
umbrella issue: https://issues.jboss.org/browse/AEROGEAR-1600

I've prepared the Community page for publishing.

We are still missing some features that I've temporarily removed:
* github activity (https://issues.jboss.org/browse/AEROGEAR-1555)
* list of contributors (https://issues.jboss.org/browse/AEROGEAR-1642)

### CODE REVIEW

**Comment by lfryc** [comment]
@corinnekrych @balunasj @gorkem @cvasilak @danbev @qmx @danielpassos @edewit @lholmquist @sebastienblanc @secondsun @andresgalante @matzew 

Guys, could you please review that you are fine with the information about you on AeroGear Team page?

http://staging-aerogearsite.rhcloud.com/community/#team

**Comment by edewit** [comment]
:+1: checked my info looks good

**Comment by matzew** [comment]
@lfryc can you remove "Specialties: iOS, WebSocket, HTML5, JMS, Stomp, AMQP, XMPP, Apache, ASF, Java EE 6, JSF 2.0, CDI, JPA, ADF Faces, Apache MyFaces, Apache MyFaces Trinidad, RIA, Rich-Internet-Applications, Server-side-push, .NET, Node.js." ? 

I think something like "Matthias is theAeroGear project lead." is good enough

File: _data/people.yml [hunk 1]
```diff
@@ -141,10 +151,11 @@ sebastienblanc:
   update-date: Wed, 17 Dec 2014 14:37:46 EST
   update-description: <a href="https://github.com/sebastienblanc">sebastienblanc</a> <span>commented</span> on pull request <a href="https://github.com/aerogear/aerogear-unifiedpush-server/pull/460#issuecomment-67167202" title="removed 'old' liquibase scripts and ...">aerogear/aerogear-unifiedpush-server#460</a>
   avatar: https://avatars2.githubusercontent.com/u/335133
-  url: sblanc.org/
-  location-city: City
-  location-country: Germany
-  description: Works at Red Hat. Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation.
+  url: sblanc.org
+  twitter: sebi2706
+  location-city:
```
Comment (sebastienblanc) [inline _data/people.yml]: You can add `Vallauris` for the city

File: _data/people.yml [hunk 2]
```diff
@@ -141,10 +151,11 @@ sebastienblanc:
...
+  location-country: France
+  description: Works at Red Hat. Sebastien leads UnifiedPush efforts in AeroGear.
```
Comment (sebastienblanc) [inline _data/people.yml]: `Sébastien`

File: _data/people.yml [hunk 3]
```diff
@@ -5,9 +5,10 @@ corinnekrych:
   update-description: <a href="https://github.com/corinnekrych">corinnekrych</a> <span>opened</span> pull request <a href="https://github.com/aerogear/aerogear-ios-oauth2/pull/17">aerogear/aerogear-ios-oauth2#17</a>
   avatar: https://avatars1.githubusercontent.com/u/1395710
   url: corinnekrych.org
-  location-city: City
```
Comment (corinnekrych) [inline _data/people.yml]: French Riviera

File: _data/people.yml [hunk 4]
```diff
@@ -5,9 +5,10 @@ corinnekrych:
...
+  twitter: corinnekrych
+  location-city:
   location-country: France
-  description: Senior technical consultant with 15 years experience, 12 companies, worked in different environments and methodologies. Enthusiastic, curious and skilled. Pragmatic thinker I will understand requirements and turn them into quality software and useful documentation.
+  description: Senior technical consultant with 15 years experience, 12 companies, worked in different environments and methodologies. Enthusiastic, curious and skilled. Pragmatic thinker I will understand requirements and turn them into quality software and useful documentation. Corinne leads iOS team in AeroGear.
```
Comment (corinnekrych) [inline _data/people.yml]: to be replaced with more friendly content:
"Code is craft and collaboration is key to success. I love chatting the latest tech trends at coffee break: female geek. Mobile is my playground, AeroGear my open source family and iOS my platform by heart."

Comment (corinnekrych) [inline _data/people.yml]: then you can leave "Corinne leads iOS team in AeroGear." as you did for all of us, but i would be nice to make 2 paragraphs as the first description uses "I" and the other "She/He"

**Comment by danielpassos** [comment]
Why us not send a PR to fix some wrong info about yourself?

**Comment by abstractj** [comment]
@sebastienblanc @corinnekrych @matzew I have to agree with @danielpassos, let's make @lfryc's life a bit easier and send PRs against his fork, once it's all about us, please.

**Comment by lfryc** [comment]
Sounds good to me!

**Comment by abstractj** [comment]
@lfryc Updated https://github.com/lfryc/aerogear.org/pull/2, let me know if I did something wrong

**Comment by matzew** [comment]
@abstractj @lfryc yeah, good idea

https://github.com/lfryc/aerogear.org/pull/3

**Comment by corinnekrych** [comment]
@lfryc just send mine too https://github.com/lfryc/aerogear.org/pull/4
Thanks!

**Comment by lfryc** [comment]
Ok, merging now, we can always change the bio later.

**Pull request merged**